src/pages/index.module.css

```
@@ -84,24 +93,105 @@
   font-size: 1.6em;
 }
 
-/* Sponsors */
-.heading {
-  text-align: center;
-  padding-bottom: 1rem;
+/* Cards */
+.cardContainer {
+  padding: 2rem 0;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 2rem;
 }
 
-.container {
+.card {
+  flex: 1 0 calc(50% - 1rem);
+  padding: 1rem;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 18rem;
+  gap: 1rem;
+}
+
+.cardSide {
+  flex: 1 0 calc(50% - 1rem);
+  height: 80%;
   display: flex;
   flex-direction: column;
-  row-gap: 5rem;
+  justify-content: center;
 }
 
-.row {
+.cardLeading {
+  align-items: flex-start;
+}
+
+.cardImage {
+  padding: 1rem;
+}
+
+.cardSpacer {
+  flex: 1;
+}
+
+@media (max-width: 800px) {
+  .card {
+    flex: 1 0 100%;
+    flex-direction: column;
+    height: auto;
+    align-items: stretch;
+  }
+
+  .cardSide {
+    flex: 0 1 auto;
+    height: auto;
+  }
+
+  .cardImage {
+    display: none;
+  }
+}
+
+/* Features and Roadmap */
+.featureRoadmapEntry {
+  flex: 0 1 max(calc(30%), 200px);
+  display: flex;
+  column-gap: 1rem;
+}
+
+.featureRoadmapIconContainer {
+  flex-shrink: 0;
+  border-radius: 50%;
+  width: 5rem;
+  height: 5rem;
+  background: var(--ifm-card-background-color);
   display: flex;
-  align-items: center;
   justify-content: center;
+  align-items: center;
+}
+
+.featureRoadmapIcon {
+  font-size: 2.5rem;
+}
+
+.featureIcon {
+  color: var(--ifm-color-primary);
+}
+
+.roadmapIcon {
+  color: var(--tauri-orange);
+}
+
+@media (max-width: 800px) {
+  .featureRoadmapEntry {
+    flex: 0 1 100%;
+  }
 }
 
-.logo {
-  margin: 1rem 0;
+/* Sponsors */
+.sponsor {
+  flex: 0 1 max(calc(25%), 200px);
+}
+
+@media (max-width: 800px) {
+  .sponsor {
+    flex: 0 1 100%;
```

Sponsor images are not centered in this case.
```suggestion
    flex: 0 1 100%;
    text-align: center;
```